tutorials-and-examples/llamaindex/rag/terraform/backend.tf

```diff
@@ -0,0 +1,6 @@
+#terraform {
```

Comment:
Should we add a comment in this file on why these are commented out?

Reply:
There is a step 3 in the section [Infrastructure Setup ](https://github.com/volatilemolotov/ai-on-gke/tree/llamaindex-guide/tutorials-and-examples/llamaindex/rag#infrastructure-setup) in the guide, which is dedicated to this. I just though that should be enough. WDYT?

Reply:
That's great, but do you mind adding a comment at the top of this file saying something like 

`# Optional gcs bucket configuration`

Thanks!